WcaOnRails/app/models/competition_event.rb

```diff
@@ -21,4 +21,31 @@ def to_wcif
       "rounds" => self.rounds.map(&:to_wcif),
     }
   end
+
+  def load_wcif!(wcif)
+    # Remove extra rounds.
+    self.rounds.each do |round|
+      wcif_round = wcif["rounds"][round.number - 1]
```

Comment:
"you only want one part and not the other" <- I think this is exactly the appeal of GraphQL. I could see us exposing WCIF as a GraphQL endpoint someday (although I would need to read up on GraphQL first). If anyone following along here has experience with GraphQL, please chime in!